introspection-engine/connectors/sql-introspection-connector/src/calculate_datamodel.rs

```diff
@@ -279,9 +411,11 @@ pub fn calculate_datamodel(
         introspection_map: &introspection_map,
     };
 
+    let introspection_map = crate::introspection_map::IntrospectionMap::new(input);
+    input.introspection_map = &introspection_map;
```

Comment:
I don't understand why we are creating the introspection map twice in this function. Maybe we should make IntrospectionMap::new()` into something that mutates the input instead?